filters/include/pcl/filters/conditional_removal.h

```diff
@@ -111,16 +111,16 @@ namespace pcl
 
     protected:
       /** \brief True if capable. */
-      bool capable_;
+      bool capable_{false};
 
       /** \brief Field name to compare data on. */
       std::string field_name_;
 
       /** \brief The data offset. */
-      std::uint32_t offset_;
+      std::uint32_t offset_{};
 
       /** \brief The comparison operator type. */
-      ComparisonOps::CompareOp op_;
+      ComparisonOps::CompareOp op_{};
```

Comment:
```suggestion
      ComparisonOps::CompareOp op_{GT};
```

Reply:
In cases like this, I don't see a reason for _any_ initialization of the member, especially not to a "random" value like `GT` (a value without any meaning or purpose). The member will be (should be) initialized elsewhere.

A situation where an initialization could make sense is when the user has to set a variable via a setter-method (like an algorithm parameter), but could possibly forget to do this. Then the variable could be initialized to an invalid value (like nan, a negative number, etc depending on the situation) and the algorithm could check whether the variable has a valid (user defined) value before using it.

Reply:
Since its an enum and don't have any "NotSet/None" value, we kinda have to set it to one of the possible values, where GT is the first defined.
Also what it currently get default initialized to.
I think many algorithms set default values for parameters, so it always run and don't thow errors for not setting specific settings, but run with a "most often used setting".
I think its a out of this PR to remedy this?